core/modules/drr.cc

```diff
@@ -70,11 +76,37 @@ CommandResponse DRR::Init(const bess::pb::DRRArg& arg) {
   }
 
   int err_num = 0;
-  flow_ring_ = AddQueue(max_number_flows_, &err_num);
+  flow_ring_ = new LockLessQueue<Flow*>(max_number_flows_, true, true);
   if (err_num != 0) {
     return CommandFailure(-err_num);
   }
 
+  /* get flow queue generator */
```

Comment:
It might help readability if you split out the code for creating a generator from `DRRArg::Codel` and `DRRArg::Llring` to different functions -- perhaps they could be in static functions in `codel.h` and `lock_less_queue.h`.